man/froll.Rd

```diff
@@ -12,71 +12,65 @@
-    calculating window? Defaults to \code{FALSE}. For details on handling
-    other non-finite values, see details below. }
-  \item{hasNA}{ logical. If it is known that \code{x} contains \code{NA}
-    then setting to \code{TRUE} will speed up. Defaults to \code{NA}. }
-  \item{adaptive}{ logical, should adaptive rolling function be
-    calculated, default \code{FALSE}. See details below. }
-  \item{FUN}{ the function to be applied in rolling fashion; see Details for restrictions }
-  \item{\dots}{ extra arguments passed to \code{FUN} in \code{frollapply}. }
+  \item{x}{ Vector, \code{data.frame} or \code{data.table} of integer, numeric or logical columns over which to calculate the windowed aggregations. May also be a list, in which case the rolling function is applied to each of its elements. }
+  \item{n}{ Integer vector giving rolling window size(s). This is the \emph{total} number of included values. Adaptive rolling functions also accept a list of integer vectors. }
+  \item{fill}{ Numeric; value to pad by. Defaults to \code{NA}. }
+  \item{algo}{ Character, default \code{"fast"}. When set to \code{"exact"}, a slower (but more accurate) algorithm is used. It
+    suffers less from floating point rounding errors by performing an extra pass, and carefully handles all non-finite values.
+    It will use mutiple cores where available. See Details for more information. }
+  \item{align}{ Character, specifying the "alignment" of the rolling window, defaulting to \code{"right"}. \code{"right"} covers preceding rows (the window \emph{ends} on the current value); \code{"left"} covers following rows (the window \emph{starts} on the current value); \code{"center"} is halfway in between (the window is \emph{centered} on the current value, biased towards \code{"left"} when \code{n} is even). }
+  \item{na.rm}{ Logical, default \code{FALSE}. Should missing values be removed when
+    calculating window? For details on handling other non-finite values, see Details. }
+  \item{hasNA}{ Logical. If it is known that \code{x} contains \code{NA}
+    then setting this to \code{TRUE} will speed up calculation. Defaults to \code{NA}. }
+  \item{adaptive}{ Logical, default \code{FALSE}. Should the rolling function be calculated adaptively? See Details below. }
+  \item{FUN}{ The function to be applied to the rolling window; see Details for restrictions. }
+  \item{\dots}{ Extra arguments passed to \code{FUN} in \code{frollapply}. }
 }
 \details{
-  \code{froll*} functions accepts vectors, lists, data.frames or
-  data.tables. They always return a list except when the input is a
-  \code{vector} and \code{length(n)==1} in which case a \code{vector}
-  is returned, for convenience. Thus rolling functions can be used
-  conveniently within data.table syntax.
+  \code{froll*} functions accept vectors, lists, \code{data.frames} or
+  \code{data.tables}. They always return a list except when the input is a
```

Comment:
I would avoid wrapping plural of data.table in code formatting. I have seen many people being confused by DataTables.

Reply:
ok, done in e35bea2